lib/sane_permalinks/sane_param.rb

```diff
@@ -10,12 +10,21 @@ def sane_param
 
   def make_param
     if(!permalink_options.empty? && !permalink_options[:prepend_id])
-      self.send(permalink_options[:with])
+      persisted_permalink_field
     elsif(!permalink_options.empty?)
-      "#{original_to_param}-#{self.send(permalink_options[:with])}"
+      "#{original_to_param}-#{persisted_permalink_field}"
```

Comment:
Apparently there is `attribute_in_database` [which already does this](https://til.hashrocket.com/posts/urdpcstonf-activerecords-attributeindatabase-method)

```suggestion
      "#{original_to_param}-#{attribute_in_database(permalink_options[:with])}"
```